app/controlplane/internal/dispatcher/dispatcher.go

```diff
@@ -184,8 +187,20 @@ func (d *FanOutDispatcher) loadInputs(ctx context.Context, queue dispatchQueue,
 		return fmt.Errorf("extracting predicate: %w", err)
 	}
 
+	// Calculate the attestation hash
+	jsonAtt, err := json.Marshal(att)
+	if err != nil {
+		return fmt.Errorf("marshaling attestation: %w", err)
+	}
+
+	h, _, err := crv1.SHA256(bytes.NewBuffer(jsonAtt))
```

Comment:
why  google/go-containerregistry here not crypto/sha256?

Reply:
Good question. I am using the google container registry one because we use it in many other places and it has this handy helper. 

That said, I might change it since it's true that it can feel out of place. Thanks!

Reply:
Actually, I just realized why I chose it. 

- To be consistent with the format of the Digest that we sent in the materials already and
- To have a decoded way to represent both the Hex and the algorithm part, separately. 

About the later, I've had many issues representing digests as strings, since sometimes they contained the algorithm and some other times they didn't.

I'd prefer then to keep it like this where the consumer of the digest can decide if they want to use one or two parts of the hash. WDYT?

Reply:
Should we add a comment then so we do not forget why? :)